server/src/test/java/org/elasticsearch/ingest/TrackingResultProcessorTests.java

```diff
@@ -17,43 +17,53 @@
  * under the License.
  */
 
-package org.elasticsearch.action.ingest;
+package org.elasticsearch.ingest;
 
 import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.action.ingest.SimulateProcessorResult;
 import org.elasticsearch.ingest.TestProcessor;
 import org.elasticsearch.ingest.CompoundProcessor;
 import org.elasticsearch.ingest.IngestDocument;
+import org.elasticsearch.ingest.TrackingResultProcessor;
 import org.elasticsearch.test.ESTestCase;
 import org.junit.Before;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.IdentityHashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 import static org.elasticsearch.ingest.CompoundProcessor.ON_FAILURE_MESSAGE_FIELD;
 import static org.elasticsearch.ingest.CompoundProcessor.ON_FAILURE_PROCESSOR_TAG_FIELD;
 import static org.elasticsearch.ingest.CompoundProcessor.ON_FAILURE_PROCESSOR_TYPE_FIELD;
-import static org.elasticsearch.action.ingest.TrackingResultProcessor.decorate;
+import static org.elasticsearch.ingest.TrackingResultProcessor.decorate;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.Matchers.nullValue;
 import static org.hamcrest.Matchers.sameInstance;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 public class TrackingResultProcessorTests extends ESTestCase {
 
     private IngestDocument ingestDocument;
     private List<SimulateProcessorResult> resultList;
+    private Set<PipelineProcessor> pipelinesSeen;
 
     @Before
     public void init() {
         ingestDocument = new IngestDocument(new HashMap<>(), new HashMap<>());
         resultList = new ArrayList<>();
+        pipelinesSeen = Collections.newSetFromMap(new IdentityHashMap<>());
     }
 
     public void testActualProcessor() throws Exception {
-        TestProcessor actualProcessor = new TestProcessor(ingestDocument -> {});
+        TestProcessor actualProcessor = new TestProcessor(ingestDocument -> {
```

Comment:
NIT: noisy reformat :)

Reply:
fixed. sorry for the noise. 
Note to self:
Intellij - Simple lambdas in one line to avoid this kind of noise.  
<img width="300" alt="image" src="https://user-images.githubusercontent.com/976291/45756922-c282ed80-bbe7-11e8-9c37-cb9167247c71.png">